src/drivers/lib.rs

```diff
@@ -7,6 +7,7 @@ extern crate common;
 extern crate hil;
 extern crate process;
 
+pub mod noop;
```

Comment:
Yeah, I see a value for a `yield` that basically context switches to other processes and renews the current process's scheduling quantum. So, e.g., I might call `yield` right before I'm about to do some CPU bound computation that I would rather complete uninterrupted.